orion/sites/admin/src/announce/announceEdit.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+require('./announceEdit.styl');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
+import { OkayModal } from '../modals/okayModal.js';
+import { YesNoModal } from '../modals/yesnoModal.js';
```

Comment:
Can remove all the API and Modal stuff now. Only include what you need.
This is important because if you're not vigilant about keeping code clean, you'll start to get a very messy codebase where you're not sure if things are still being used or not.